dex/networks/eth/tokens.go

```diff
@@ -173,20 +173,119 @@ var Tokens = map[uint32]*Token{
 			},
 		},
 	},
+	usdtTokenID: {
+		EVMFactor: new(int64),
+		Token: &dex.Token{
+			ParentID: EthBipID,
+			Name:     "USDT",
+			UnitInfo: dex.UnitInfo{
+				AtomicUnit: "microUSD",
+				Conventional: dex.Denomination{
+					Unit:             "USDT",
+					ConversionFactor: 1e6,
+				},
+			},
+		},
+		NetTokens: map[dex.Network]*NetToken{
+			dex.Mainnet: {
+				Address: common.HexToAddress("0xdac17f958d2ee523a2206206994597c13d831ec7"), // https://etherscan.io/address/0xdac17f958d2ee523a2206206994597c13d831ec7
+				SwapContracts: map[uint32]*SwapContract{
+					0: {
+						// swap contract: https://etherscan.io/address/0x97a53fEF7854f4CB846F2eaCCf847229F1E10e4f
+						Address: common.HexToAddress("0x97a53fEF7854f4CB846F2eaCCf847229F1E10e4f"),
+						// USDT's contract is upgradable, using a proxy call, so
+						// gas cost could change without notice, so we do not
+						// want to set limits too low, even with live estimates.
```

Comment:
It looks like you're just using 30% buffer from getgas. Are you suggesting we should add more?

Reply:
No, I think it's fine. This comment was just copied over from USDC.